src/classes/LGTV.ts

```diff
@@ -58,6 +58,23 @@ export class LGTV {
     return await this.sendCommand(`POWER off`);
   }
 
+  async launchApp(name: Apps) {
```

Comment:
I wonder if it'd be worth allowing any values here, given applications will certainly change often and shouldn't be referenced in code.

```suggestion
  async launchApp(name: Apps | string) {
```

Reply:
I think it's nice to enumerate all the major ones, to make this library easier to use by other projects (such as [Companion](https://github.com/llamafilm/companion-module-lgtv-display/tree/feature/launchApp)).  Hopefully they don't change often.

Reply:
Yes, but we should also allow other options to be set, that's why I'm suggesting `| string`.